setup/tools/sqlgen/source.func.php

```diff
@@ -1045,9 +1045,9 @@ function taughtSpell($item)
     CLISetup::log('   * #4  Quest');
     $quests = DB::World()->select('
         SELECT spell AS ARRAY_KEY, id, SUM(qty) AS qty, BIT_OR(side) AS side FROM (

```

Comment:
you'll probably want to also change `id` to uppercase here and consequently in line 1065.
But i'm not sure how strict mysql goes about that.